.vscode/launch.json

```diff
@@ -9,7 +9,18 @@
             "type": "python",
             "justMyCode": false,
             "request": "launch",
-            "name": "VehicleApp",
+            "name": "VehicleApp (native middleware)",
```

Comment:
just to have it on your mind: this also need to be touched in regards of moving to native:
https://github.com/eclipse-velocitas/vehicle-app-python-sdk/blob/main/.project-creation/templates/.vscode/launch.json